.github/workflows/automerge.yml

```diff
@@ -0,0 +1,15 @@
+name: Auto-merger
+
+on:
+  pull_request:
+
+jobs:
+  optimize-auto-merge:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.title == 'Optimize SVG(s)'
+    steps:
+      - env:
+          PR_URL: ${{ github.event.pull_request.html_url }}
+        run: |
+          gh pr comment $PR_URL --body "No issues found, auto-merging the pull request."
+          gh pr merge $PR_URL --auto --squash --delete-branch
```

Comment:
Might be a problem, because we would need to let it run a bit for the required check to complete then we can merge it.